manifests/08-clusteroperator.yaml

```diff
@@ -3,3 +3,9 @@ kind: ClusterOperator
 metadata:
   name: console
 spec: {}
+status:
+  versions:
+    - name: operator
+      version: "0.0.1-snapshot"
+    - name: openshift-console
+      version: "0.0.1-snapshot_openshift"
```

Comment:
This would need to come out of the YAML manifest if we're not yet reporting console version